shared/AppInsightsCommon/src/Offline.ts

```diff
@@ -20,7 +49,13 @@ export function createOfflineListener(parentEvtNamespace?: string | string[]): I
     let _document = getDocument();
     var _navigator = getNavigator();        // Gets the window.navigator or workerNavigator depending on the global
     let _isListening: boolean = false;
-    let _onlineStatus: boolean = true;
+    let listenerList: OfflineCallback[] = [];
+
+    // Set the initial state
+    let _onlineStatus: boolean = _isNavOnline();
```

Comment:
the `_onlineStatus` IS the runtime state.

Reply:
Looking at the reset of the code, it looks like you can change this to be called `rState` and remove the one below.

Reply:
Ok, finally finished reading through the PR.

This should be called `_currentState` as you are recording / updating this in the current check.
It should also be "set" (by calling the `currentState()`) after you set the `rState` and `uState`

Reply:
Will it be a boolean or eOfflineValue? As rState is a eOfflineValue while _onlineStatus is a boolean.

Reply:
got it~ it will be set in listnerNoticeCheck